Sources/Common/Errors.swift

```diff
@@ -224,16 +224,17 @@ public struct SecurityError: Error {
 
 extension SecurityError: CustomStringConvertible {
     static var nilPublicKey = """
-        the public key could not be extracted (this can happen if the public key algorithm is not supported).
+        the public key could not be extracted
```

Comment:
Did you want to change this to a multi-line error?  If not just stick the "\" at the end of the first line.